src/components/addDoctor/AddDoctor.js

```diff
@@ -0,0 +1,154 @@
+import { React, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import swal from 'sweetalert';
+import { useNavigate } from 'react-router-dom';
+import './addDoctor.css';
+import { getDoctors, postData } from '../../redux/doctors/doctorSlice';
+
+export default function AddDoctor() {
```

Comment:
- [ ] After adding a doctor, it does not appear on the "doctors" section unless I reload the page.

A new request to the server needs to be done each time a doctor is either added or deleted.

That way, your variables will be updated, and your UX improved. 👍 

- [ ] After deleting a doctor, it stil appears on the "doctors" section unless I reload the page.

A new request to the server needs to be done each time a doctor is either added or deleted.

That way, your variables will be updated, and your UX improved. 👍